recipes/gn/all/conandata.yml

```diff
@@ -1,3 +1,4 @@
 sources:
   "cci.20210429":
     url: "https://gn.googlesource.com/gn/+archive/6771ce569fb4803dad7a427aa2e2c23e960b917e.tar.gz"
+    sha256: "d05eb257ebf9aeaf9238346624f2a30e91e1e7a0ca78eb0e0e42274591338443"
```

Comment:
This hash is not stable, I just downloaded the archive and got `80bcb40e082489588b9572082116ce2c1f18a0a7f89c7d63173e250002a57ccf` now.

Reply:
Perhaps add a comment about it not being stable to avoid future attempts re-adding it.

Reply:
Ok, so we need to add it as exception to the hook

Reply:
Ok, so we need to add it as exception to the hook